lib/translator/inquiry.cc

```diff
@@ -171,9 +171,9 @@ void TranslateBlockLimitsVpd(const nvme::IdentifyControllerData& identify_ctrl,
   // TODO: put in common?
   const uint8_t kMaxCompareWriteLen = 255;
   // compare and write length is 8 bits. we ensure the size fits.
-  uint32_t compare_and_write_len = (max_transfer_length > kMaxCompareWriteLen)
-                                       ? kMaxCompareWriteLen
-                                       : max_transfer_length;
+  uint8_t compare_and_write_len = (max_transfer_length > kMaxCompareWriteLen)
```

Comment:
As per the comment in line 173, this field is supposed to be 8 bits. I made the change to get rid of a compiler narrow conversion warning